assets/translations/en-us.yaml

```diff
@@ -5323,13 +5324,23 @@ harvester:
         anyNode: 'Run VM on any available node'
         schedulingRules: 'Run VM on node(s) matching scheduling rules'
         specificNode: Run VM on specific node(s) - (Live migration is not supported)
+    accessCredentials:
+      resetPwd:
+        label: Add Basic Auth
+      injectSSH:
+        label: Add SSHKey
+        users: Select Users
+        addUser: Add User
+      tips: qemu-guest-agent must be installed to enable Access Credentials, the VM should be restarted after editing/deleting credentials.
```

Comment:
@guangbochen Is this a authentic translation?

Reply:
Yep, also from the UI side, make sure we disable the `set credentials` action when no `qemu-agent` is detected and show the notification message accordingly.

Reply:
The credentials management page needs to be displayed (to let the user know the current configuration), but can't be added/edited/removed without qemu-guest-agent (because it won't work).